pkg/printer/pushgateway.go

```diff
@@ -0,0 +1,85 @@
+/**
+ * Copyright 2022 Cisco and its affiliates
+ * All rights reserved.
+**/
+
+package printer
+
+import (
+	"fmt"
+	"io"
+
+	config "github.com/get-woke/woke/pkg/config"
+	"github.com/get-woke/woke/pkg/result"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/push"
+)
+
+// Pushgateway is a way to push metrics within prometheus
+type Pushgateway struct {
+	writer         io.Writer
+	labels         map[string]string
+	pushgatewayURL string
+}
+
+// NewPushgateway returns a Pushgateway Printer with color optionally disabled
+func NewPushgateway(w io.Writer, c *config.Config) *Pushgateway {
+	return &Pushgateway{
+		writer:         w,
+		labels:         c.Labels,
+		pushgatewayURL: c.PushgatewayURL,
+	}
+}
+
+func (t *Pushgateway) PrintSuccessExitMessage() bool {
+	return true
+}
+
+// Print prints the file results
```

Comment:
This comment should be updated to be more descriptive, this looks like a copy/paste from text.go. Links/explanations for this printer would be helpful.